subjects/Hunk-m68k/BENCHLNG.reko/BENCHLNG_code.c

```diff
@@ -23,12 +23,12 @@ void fn00001000(int32 d0, byte * a0)
 	globals->ptr3E20 = fp;
 	globals->ptr3E24 = a6_n;
 	struct Eq_n * d0_n = FindTask(0x00);
-	Eq_n a1_n = 0x12BC;
-	Eq_n d0_n = OpenLibrary(0x12BC, 0);
-	if (d0_n != 0x00)
+	struct Eq_n * a1_n = &globals->t12BC;
+	struct Eq_n * d0_n = OpenLibrary(0x12BC, 0);
```

Comment:
This a note to myself: `OpenLibrary` should have enough type info to resolve this.